var/spack/repos/builtin/packages/py-ww/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyWw(PythonPackage):
+    """Wrappers for Python builtins with higher-level APIs."""
+
+    homepage = "https://github.com/tygs/ww/"
+    pypi     = "ww/ww-0.2.1.tar.gz"
+
+    version('0.2.1', sha256='3664f1f91bf927fe597ab153e8df73c8954927258b3737220efd1cb9912ebd7e')
+
+    depends_on('py-setuptools', type='build')
```

Comment:
`py-pytest-runner` is also listed as a build dep in the setup.py

Reply:
Thanks! It's added now.